wagtail_localize_rws_languagecloud/emails.py

```diff
@@ -13,22 +14,73 @@ def get_full_url(url):
     return url
 
 
-def send_emails(translation):
-    subject, body = compose_email(translation)
+# sync_rws command emails
+def send_sync_rws_emails(translation):
+    subject, body = compose_sync_rws_email(translation)
     recipients = get_recipients()
     for recipient in recipients:
         send_mail(subject, body, [recipient])
 
 
-def compose_email(translation):
+def compose_sync_rws_email(translation):
     subject = gettext_lazy("Translated content ready for review")
     body = gettext_lazy(
         "Translated content for '%(instance)s' is ready for review at: %(edit_url)s"
-        % {
-            "instance": str(translation.get_target_instance()),
-            "edit_url": get_full_url(translation.get_target_instance_edit_url()),
+    ) % {
+        "instance": str(translation.get_target_instance()),
+        "edit_url": get_full_url(translation.get_target_instance_edit_url()),
+    }
+    return subject, body
+
+
+# update_translated_pages emails
+def send_update_translated_pages_emails(all_pages, updated_pages, skipped_pages):
+    """Notifies users that translated pages have been synced.
+
+    Args:
+        all_pages (List[Tuple[int, str]]): List of all pages considered.
+        updated_pages (List[Tuple[int, str]]): List of updated pages.
+        skipped_pages (List[Tuple[int, str]]): List of pages that were not updated.
+    """
+    subject, body = compose_update_translated_pages_email(
+        all_pages, updated_pages, skipped_pages
+    )
+    recipients = get_recipients()
+    for recipient in recipients:
+        send_mail(subject, body, [recipient])
+
+
+UPDATE_TRANSLATED_PAGES_EMAIL_TEMPLATE = Template(
+    """{% load i18n %}
+{% blocktrans with total_count=all_pages|length updated_count=updated_pages|length skipped_count=skipped_pages|length %}
+Found {{ total_count }} page(s) with stale translations, of which {{ updated_count }} were synced and {{ skipped_count }} were skipped.{% endblocktrans %}
+
+{% trans "Pages that were synced:" %}
+
+{% for page_pk, page_title in updated_pages %}{{ forloop.counter }}. {{ page_title }}: {% url 'wagtailadmin_pages:edit' page_pk %}{% empty %}-
+{% endfor %}
+
+{% trans "Pages that were skipped because they had pending translations on RWS LanguageCloud:" %}
+
+{% for page_pk, page_title in skipped_pages %}{{ forloop.counter }}. {{ page_title }}: {{ base_url }}{% url 'wagtailadmin_pages:edit' page_pk %}{% empty %}-{% endfor %}
+"""
+)
```

Comment:
Not a blocker for merging, but this is a bit gross IMO. Fine for a short template but for one this big/complex I think we could bung it in a file and do:

```python
from django.template import loader
loader.render_to_string('path/to/file.html', context)
```

Reply:
I was torn on doing this, but I'll make this change.

Reply:
I've extracted the email text to its own file.